tools/Archiver/project.json

```diff
@@ -16,5 +16,18 @@
   },
   "frameworks": {
     "netcoreapp1.0": {}
+  },
+  "runtimes": {
+    "win7-x64": {},
+    "win7-x86": {},
+    "osx.10.10-x64": {},
+    "osx.10.11-x64": {},
```

Comment:
You don't need osx.10.10, as we don't really support it.

Here's the list from our dotnet-cli-build\project.json:

    "win7-x64": {},
    "win7-x86": {},
    "osx.10.11-x64": {},
    "ubuntu.14.04-x64": {},
    "ubuntu.16.04-x64": {},
    "centos.7-x64": {},
    "rhel.7.2-x64": {},
    "debian.8-x64": {},
    "fedora.23-x64": {},
    "opensuse.13.2-x64": {}